FAQ.md

```diff
@@ -58,112 +50,81 @@ types of programming languages.
 
-### Ambassador license
+#### How are modifications to Slint itself covered under this licence?
 
-#### Why is the Ambassador license free-of-charge?
+The license does not restrict 'if' and 'how' the modifications to Slint should be distributed. Say for example, Alice uses Slint under this new license to develop application A and modifies Slint in some way. She may choose to release the modifications to Slint under any license of her choice including any of the open source licenses. Alternatively she may decide not to release the modifications.
 
-The license is provided free-of-charge to achieve the following goals:
+#### If Slint were to be taken over by a larger company or the current owners were to have a change of heart, can they revoke existing licenses?
 
-a. accelerate adoption of Slint,
-b. allow developers to use Slint under a non-GPL license,
-c. create a strong feedback loop to improve Slint.
+The commitment to provide Slint under a Royalty-free license has been included in the [Contributors License Agreement (CLA)](http://cla-assistant.io/slint-ui/slint) and thereby with the larger Slint community.
 
-#### When does the Ambassador license run out?
+### GPLv3
 
-The license grant is [perpetual](#what-does-perpetual-mean), which means that you can use Slint free-of-charge forever. The perpetual nature of the license also implies that even if we need to modify the terms of the license in the future, the modified terms will not apply to already granted licenses.
+#### If I link my program with Slint GPLv3, does it mean that I have to license my program under the GPLv3, too?
 
-#### For how long do you plan to offer the free-of-charge Ambasssador license?
+No. You can license your program under any license compatible with the GPLv3 such as [https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses).
 
-Forever. However, we may modify the terms of the license in the future based on user feedback and business needs.
+Refer to GPL FAQ [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL).
 
-#### How can I get the Ambassador license?
+#### My MIT-licensed program links to Slint GPLv3. Can someone fork my program to build and distribute a proprietary program?
 
-The license is automatically granted on signing the license agreement [here](https://slint-ui.com/ambassador-program.html#application). The authorized signatory could be the code owner, primary maintainer, or in case of an organisation, the relevant person authorized to sign contracts on behalf of the organisation.
+Yes, provided the person distributing the proprietary program acquired a Slint proprietary license instead of using Slint under GPLv3, or removed the dependency to Slint altogether.
```

Comment:
Updated to 
Yes, provided the person distributing the proprietary program acquired a Slint proprietary license, such as the Slint Royalty-free license or a paid license, instead of using Slint under GPLv3. The other option would be to remove the dependency to Slint altogether.